output.tf

```diff
@@ -0,0 +1,36 @@
+output "organization" {
+  value = aws_organizations_organization.this.id
+}
+
+output "enabled_regions" {
+  value = var.config.enabled_regions
+}
+
+output "break_glass_access" {
+  value = [for username in var.config.break_glass_accounts : {
+    username = username,
+    password = aws_iam_user_login_profile.break_glass_access[username].password
```

Comment:
Should we not store this as a SSM parameter instead??

Reply:
No - this is by design. Break glass access is intended for when there is no other access to the account including ssm.